test/AutoRest.TestServer.Tests/Mgmt/OutputLibrary/OutputLibraryTestBase.cs

```diff
@@ -115,14 +115,6 @@ private ConstructorInfo GetLeastParamCtor(Type generatedModel)
             return leastParamCtor;
         }
 
-        [Test]
-        public void ValidateResourceDataCount()
-        {
-            var count = MgmtContext.Library.ResourceSchemaMap.Value.Count;
-
-            Assert.AreEqual(count, MgmtContext.Library.ResourceData.Count(), "Did not find the expected resourceData count");
-        }
```

Comment:
well is it possible that we could keep this?

Reply:
The count doesn't match anymore

Reply:
Or, it can be ResourceCount = ResourceDataCount + PartialResourceCount